test/runtime-integration/expected-cjs-22.0.txt

```diff
@@ -0,0 +1,10 @@
+================= require - auto ====================
```

Comment:
The require_esm warning message was introduced in Node.js 22.11, so it should be renamed.

We can also add a new test for Node.js 22.10, that does not emit such warning.

Reply:
https://nodejs.org/en/blog/release/v22.11.0
It seems that v22.11.0 introduced no new changes?

Reply:
Oh it was introduced in Node.js 22.10: https://nodejs.org/en/blog/release/v22.10.0

So we can add a new test for 22.9 and 22.10 then.

Reply:
https://github.com/babel/babel/actions/runs/11623032917/job/32369463734
It seems that `require(esm)` is not enabled by default, should we enable it manually? Or wait for the backport to v22 before adding it.

Reply:
Oh my bad, I thought the ERR_REQUIRE_ESM error as experimental warnings.